tasmota/tasmota_xdrv_driver/xdrv_03_esp32_energy.ino

```diff
@@ -891,12 +891,11 @@ void EnergyEverySecond(void) {
           Energy->voltage[i] = 0;
           Energy->current[i] = 0;
           Energy->active_power[i] = 0;
-          if (!isnan(Energy->apparent_power[i])) { Energy->apparent_power[i] = 0; }
-          if (!isnan(Energy->reactive_power[i])) { Energy->reactive_power[i] = 0; }
-          if (!isnan(Energy->frequency[i]) && !Energy->type_dc) { Energy->frequency[i] = 0; }
-          if (!isnan(Energy->power_factor[i])) { Energy->power_factor[i] = 0; }
-          if (!isnan(Energy->export_active[i])) { Energy->export_active[i] = 0; }
-
+          if (!isnan(Energy->apparent_power[i])) Energy->apparent_power[i] = 0;
```

Comment:
Why oh why removing my favorite brackets???? Pls reinstall asap

Reply:
Your loved brackets are back. 😉
For me no brackets looking better in one-line-statement, but in future I'll place them.